src/main/kotlin/pt/up/fe/ni/website/backend/config/CorsConfig.kt

```diff
@@ -0,0 +1,31 @@
+package pt.up.fe.ni.website.backend.config
```

Comment:
Just feels a bit weird having this at the root of config, since we have packages for other configs. I'll suggest moving this into `config/auth` and renaming the package to something like `security`